docs/docs/run_minder_server/config_provider.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Configure a Provider
+sidebar_position: 20
+---
+
+# Configuring a Provider
+A provider connects Minder to your software supply chain — giving Minder information about your source code repositories, and their pull requests, dependencies, and artifacts.
+
+The currently supported providers are:
+- GitHub   
+
+For GitHub, you configure a Provider by creating a GitHub App. 
+
+## Prerequisites
+
+- [GitHub](https://github.com) account
+
+## Create a GitHub App
+This approach allows users fine-grained control over the permissions that Minder has in their repositories. It also 
```

Comment:
```suggestion
This approach allows fine-grained control over the permissions that Minder has in users' repositories.  It also 
```

Users can accept or reject the permissions of a GitHub App, but they can't change or control them, except by selecting which organizations and repos the App has access to.  The permissions _are_ fine-grained, but the control is from the application author side (because it would be too hard for app authors to code for every combination of permissions).

Reply:
Thanks for clarifying that. This was a copy/paste of the old docs, but I wasn't sure of the correctness.